envelop/__init__.py

```diff
@@ -17,9 +17,10 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from __future__ import unicode_literals
+from six.moves.urllib.parse import urlparse
+from builtins import str as text
```

Comment:
Python 2.x doesn't have the module `builtins`, we should probably have something from six (`six.text_type` maybe?) or define it ourselves.